### ISSUE TITLE
Allow comments in DDLog interpreter

### ISSUE THREAD

**Issue by lykahb** [vmware/differential-datalog#1057]
For better readability I'd like to be able to leave comments in the .dat files. Now the interpreter rejects the comments as an invalid syntax.

I have a workaround but it's rather ugly and doesn't work for the trailing comments.
```
test_ddlog/target/debug/test_cli < <(cat init.dat | grep -v "^//")
```

**Comment by ryzhyk** [comment]
Lines starting with '#' are interpreted as comments by ddlog in  command files.

**Comment by lykahb** [comment]
Thank you, that works.

Both Rust and Datalog use `//` for comments. Also, the top example in command_reference.md uses `//` for comments. So I didn't think of the alternatives. 

I opened a PR https://github.com/vmware/differential-datalog/pull/1058 with a fix for the examples.

**Comment by ryzhyk** [comment]
Resolved via #1058.
